index.html

```diff
@@ -0,0 +1,228 @@
+              />
+              <path
+                d="M2238.4,530.6c0-0.5,0-0.9-0.5-1.9l-0.5-1.9l-9.5-38.9c-0.5-1.9-1.9-2.8-3.8-2.8c-0.5,0-1.9,0-3.3,0.5  c-6.2,1.9-10.5,2.8-15.7,2.8c-6.7,0-10-2.8-11.9-9.5c-0.9-3.8-0.9-4.8-0.9-31.3V200.2c0-5.7-1.4-8.1-4.8-8.1c-1,0-1,0-5.7,1  l-42.8,8.5c-4.8,0.9-7.1,3.8-7.1,9v241.2c0,41.3,0.5,47,4.3,57.9c7.6,22.3,26.6,34.2,54.2,34.2  C2213.2,543.9,2238.4,536.8,2238.4,530.6z"
+              />
+              <path
+                d="M2337.5,231.5c0-20.9-17.1-38.5-38-38.5c-21.4,0-39,17.1-39,38c0,21.8,17.1,38.9,38.5,38.9  C2320.4,270,2337.5,252.9,2337.5,231.5z M2330.9,527.8V309.4c0-7.1-0.9-8.1-8.1-8.1h-88.4c-7.1,0-8.1,0.9-8.1,8.1v28  c0,7.1,1,8.1,8.1,8.1h36.1v182.3c0,7.1,0.9,8.1,8.1,8.1h44.2C2329.9,535.9,2330.9,534.9,2330.9,527.8z"
+              />
+              <path
+                d="M2535.6,320.3c0-11.9-38.5-27.1-67.5-27.1c-67.9,0-109.8,49.4-109.8,130.1c0,76.4,37.5,120.6,102.2,120.6  c32.3,0,68.9-13.8,68.9-25.6c0-1.9,0-1.9-1.4-5.7l-11.4-29c-0.9-1.9-2.4-3.3-3.8-3.3c-1.4,0-3.3,0.9-5.2,2.4  c-13.8,9-25.7,13.3-39,13.3c-32.8,0-47.5-23.3-47.5-75c0-55.1,14.7-79.8,47.1-79.8c14.3,0,22.8,3.3,40.4,16.1  c1.9,1.4,3.3,1.9,4.8,1.9c1.4,0,3.3-1.4,4.3-3.3l16.6-30.4C2535.2,323.6,2535.6,321.7,2535.6,320.3z"
+              />
+              <path
+                d="M2761,515.4c0-1.4-0.9-3.8-2.9-5.7l-87.4-105.4l77.9-75c2.4-2.4,3.3-4.3,3.3-5.7c0-1.9-1.4-3.8-3.8-5.7l-25.7-20.4  c-3.8-3.3-3.8-3.3-5.7-3.3c-1.9,0-1.9,0-5.7,3.8l-92.2,92.6V200.2c0-5.7-1.4-8.1-4.8-8.1l-4.3,0.5l-44.2,9c-6.6,1.4-7.1,2.4-7.1,8.5  v317.6c0,7.1,0.9,8.1,8.1,8.1h44.2c7.1,0,8.1-0.9,8.1-8.1V425.7l88.9,113.9c2.4,3.3,3.8,4.3,6.2,4.3c1.4,0,1.4,0,4.8-1.9l37.5-20.9  C2759.6,519.2,2761,517.3,2761,515.4z"
+              /></svg
+          ></a>
+          <!-- VS breaks the SVG-Code, so the following code seems broken. When you set the last break in the closing tag </svg> in one line, everything is o.k., but after save, it's broken again -->
+          <a href="#" class="standard-link-item standard-link-item-1"
+            >Für euch</a
+          >
+          <a href="#" class="standard-link-item standard-link-item-2"
+            >Für Unternehmen</a
+          >
+          <a href="#" class="standard-link-item standard-link-item-3"
+            >Fürs Klima</a
+          >
+        </nav>
+      </section>
+    </header>
+    <header class="header-image">
+      <figure>
+        <img
+          src="./assets/img/pexels-roberto-nickson-2559941.jpg"
+          srcset="
+            ./assets/img/pexels-roberto-nickson-2559941-mobil.jpg 767w,
+            ./assets/img/pexels-roberto-nickson-2559941.jpg       800w
+          "
+          alt="Landschaft"
+        />
+      </figure>
+      <section class="bck-standard-gray header-teaser-box">
```

Comment:
Prüfe bitte ob `article` in `section` semantisch überhaupt Sinn ergibt.

Es ist keine Schande `div` Elemente zu verwenden, wenn man block-level Element fürs Styling braucht.